include/mgp.hpp

```diff
@@ -2723,6 +2838,43 @@ inline bool Path::operator==(const Path &other) const { return util::PathsEqual(
 
 inline bool Path::operator!=(const Path &other) const { return !(*this == other); }
 
+inline const std::string Path::ToString() const {
+  const auto length = Length();
+  size_t i = 0;
+  std::string return_string = "";
+  for (i = 0; i < length; i++) {
+    std::string labels{""};
+    const auto node = GetNodeAt(i);
+    for (auto label : node.Labels()) {
+      labels += ":" + std::string(label) + " ";
+    }
+    return_string += "(id: " + std::to_string(node.Id().AsInt()) + " " + labels + ")-";
```

Comment:
Can you use `node.ToString()` here?

Reply:
I think it is not that of a problem if you use it here, it should be okay that we get all the labels and all the properties of node